style.css

```diff
@@ -87,6 +91,22 @@ a {
 :root .has-dark-alt-color {
 	color: var( --color-dark-alt-text );
 }
+:root .has-dark-alt-background-color {
+	background-color: var( --color-dark-alt-text );
+}
+:root .has-accent-background-color {
+	color: var( --color-accent-background );
+	background-color: inherit;
```

Comment:
Should the background-color be set to `var( --color-primary-text )` or something explicit? 

For example, when I set a button's text color to `Accent Background`, the button appears with the yellow against a transparent background. I guess this is better than the 2020 theme accent color as the background color, but not sure if it's fully the intention.

Reply:
@jffng - No. I (confusingly) named a color `accent-background`, so it actually doesn't include the background color. The selector to set the accent background is actually `has-accent-background-background-color`.

However, because the parent theme is somewhat saner, it sets the background color for `accent` and I need to reset it here.

I'll make a ticket for renaming colors to have a bit more sanity, but that should be after this is merged imo.

Reply:
Gotcha. I guess what I'm suggesting is, I would expect the background colors to be the same if a text color is selected and no background color is selected, but this is what I'm getting at the moment using buttons as an example, one for each text color:  

<img width="703" alt="Screen Shot 2020-04-28 at 11 21 28 PM" src="https://user-images.githubusercontent.com/5375500/80558082-0a165180-89a7-11ea-8f05-51d8d31466e8.png">

Reply:
Oh, interesting! I'll see if I can resolved that.

Reply:
I think this should be resolved by #20, as then with a new name the code won't need to set `inherit` here.